runtime/src/main/java/org/corfudb/runtime/view/SequencerView.java

```diff
@@ -30,13 +30,13 @@ public SequencerView(CorfuRuntime runtime) {
      * @param numTokens The number of tokens to reserve.
      * @return The first token retrieved.
      */
-    public TokenResponse nextToken(Set<UUID> streamIDs, int numTokens) {
+    public TokenResponse nextToken(List<UUID> streamIDs, int numTokens) {
```

Comment:
There's many single tail query calls in the code, it will be cleaner to introduce a new method public TokenResponse queryTail(UUID streamID, int numTokens)